datadome-client-go/client.go

```diff
@@ -39,8 +39,9 @@ func (c *Client) doRequest(req *http.Request, httpResponse *HttpResponse) (*Http
 	// Add apikey as a query parameter on each request for authentication
 	// Add also withoutTraffic parameter to true to have better performances
 	q := req.URL.Query()
-	q.Add("apikey", c.Token)
-	q.Add("withoutTraffic", "true")
+	//q.Add("apikey", c.Token)
+	req.Header.Set("x-api-key", "value")
+	q.Add("withoutTraffic", c.Token)
```

Comment:
That doesn't seems ok. Why "value" and why do you send the token in the query param without Traffic ?